RELEASE-NOTES.md

```diff
@@ -1,3 +1,33 @@
+### 0.11.0
+- Added virtual machine with automating configuration of golem and concent for testing purpose (#146, #147, #148, #182, #186).
+- Added liveness and readiness probe to geth (#44).
```

Comment:
We have added and removed the readiness probe before the code was released. It should not be mentioned. Mention only the liveness probe and link to both issues.